.zenodo.json

```diff
@@ -0,0 +1,23 @@
+{
+    "description": "A set of tools for numerically solving the nonlinear magnetic induction equation with Hall effect in OpenCL", 
+    "license": "other-open", 
+    "title": "MuMPlaCL/InductionEq", 
+    "version": "v1.1.1", 

```

Comment:
Do we need to specify the version number (and the publication date given below)? What happens if we do not give that information? Does Zenodo use the latest release date and version number? If yes, I would prefer to have it that way (since we would not need to keep this file up to date for every release).

Reply:
At some point dynamic version numbers and dates should become available in conjunction with json config files, but i does not work right now.

Reply:
Okay.